src/org/ods/services/GitService.groovy

```diff
@@ -207,6 +207,40 @@ class GitService {
         return branchCheckStatus == 0
     }
 
+    boolean otherFilesChangedAfterCommitOfFile(String fileName, String openshiftDir) {
+        if (!script.fileExists(fileName)) {
+            return true
+        }
+        try {
+            def commitOfFile = script.sh(
+                script: """git log -1 --format=%H ${fileName}""",
+                    returnStdout: true,
+                    label: "Get commit of ${fileName}"
+                ).trim()
+            if (commitOfFile) {
+                def filesChanged = script.sh(
+                    script: """git diff ${commitOfFile} HEAD --name-only""",
+                        returnStdout: true,
+                        label: "Get changes after commit ${commitOfFile}"
+                    ).trim()
+                List files = filesChanged.normalize().readLines()
+                // remove with ${} does not work .. wtf..
+                def templateYml = openshiftDir + '/template.yml'
+                files.remove(templateYml)
```

Comment:
Oh Groovy. I tried to figure out why but couldn't. `files.removeAll{ it =="${openshiftDir}/template.yml" )` would work though.